CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Added
 
+- [#3181](https://github.com/thanos-io/thanos/pull/3181) Query Frontend: Add debug level logging for responses between 300-399
```

Comment:
It is not specific to `Query Frontend` component. This change is for the logging library and I think all components with an HTTP server are using it. @yashrsharma44 Hi, can you please help confirm this?

Reply:
Yeah, it is not a `Query Frontend` component issue. 
@aribalam can you change the component to `Logging`, because the change affects all the components which use logging.

Reply:
Changed! :)